microsoft/testsuites/core/provisioning.py

```diff
@@ -191,6 +193,31 @@ def verify_reboot_in_platform(
             case_name="verify_reboot_in_platform",
         )
 
+    @TestCaseMetadata(
+        description="""
+        This case runs smoke test on a node provisioned with ultra disk.
+        The test steps are same as `smoke_test`.
+        """,
+        priority=3,
```

Comment:
Does it make sense to have priority different than other provisioning tests?

Reply:
@johnsongeorge-w any thoughts on this?

Reply:
It was a copy paste error. Changed it to 1.